.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: build
+on:
+  # run on push but only for the master branch
+  push:
+    branches:
+      - master
+  # run for every pull request
+  pull_request: {}
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10.13, 12, 14]
+    steps:
+      - name: 🛑 Cancel Previous Runs
```

Comment:
Do we need this step? Sometimes it's weird if previous commits get a red dot, even if they would technically pass.

Reply:
I was researching this and found out that Travis CI does has this behavior by default (kill any previously running CI jobs for PRs when updates come in). I assumed we wanted it as well and added this step for it. 

Here's an example of this happening on this PR when I pushed a commit and then pushed another commit a couple minutes later: https://github.com/paypal/paypal-checkout-components/actions/runs/406899687

I agree that it's annoying to see red even though it's not a failure. The nice thing about canceling previous runs is we free up ci resources. 

I'll remove this and if it becomes a problem then we can add it back in.